index.html

```diff
@@ -4,12 +4,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="./styles/reset.css" rel="stylesheet" type="text/css" />
+    <link href="./styles/reset.css" rel="stylesheet">
+    <link href="./styles/style.css" rel="stylesheet">
+    <link href="./styles/button.css" rel="stylesheet">
+    <link href="./styles/text.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css" />
     <title>밤돌이로</title>
 </head>
 <body>
     <main>
-        <!-- 코드를 작성해주세요. -->
+        <div class="container">
+            <img src="img/bssm.png"style="width:100%;">
+                <div id="top">
+                    <img class="logo" src="img/logo.png">
+                    <div class="btn1">
+                        <ul>
+                            <li><p class="login">로그인</p></li>
+                            <li><button id="join">회원가입</button></li>
+                        </ul>
+                    </div>
+                    <div class="square"></div>
```

Comment:
안에 아무 요소도 없다면
```suggestion
                    <div class="square" />
```

이런식으로 쓰는것은 어떨까요?